tree/ntuple/v7/inc/ROOT/RNTupleUtil.hxx

```diff
@@ -27,6 +27,10 @@
 
 namespace ROOT {
 
+class RLogChannel;
+/// Log channel for RNTuple diagnostics.
+ROOT::RLogChannel &NTupleLog();
+
```

Comment:
Actually, do we want this in the public interface? Would it be sufficient to be in `ROOT::Internal`?